packages/cli/src/scaling/types.ts

```diff
@@ -53,3 +53,14 @@ export type RunningJob = {
 };
 
 export type RunningJobSummary = Omit<RunningJob, 'run'>;
+
+export type QueueRecoverySettings = {
+	/** ID of timeout for next scheduled recovery cycle. */
+	timeout?: NodeJS.Timeout;
```

Comment:
The prefix `Settings` implies that this would contain only configuration values. `timeout` however is dynamic state of the process. Could we either move it out from the type or rename the type to something else, e.g. `QueueRecoveryContext`

Reply:
[7b47c75](https://github.com/n8n-io/n8n/pull/10368/commits/7b47c752079f99005c22f4dc959661fb23e8d506)